packages/esm-forms-workspace-app/src/config-schema.ts

```diff
@@ -0,0 +1,81 @@
+import { validator, Type } from '@openmrs/esm-framework';
+
+export const configSchema = {
+  formsSectionsConfig: {
+    _type: Type.Array,
+    _elements: {
+      name: {
+        _type: Type.String,
+        _description: 'Name of the section.',
+      },
+      labelCode: {
+        _type: Type.String,
+        _description: 'Label to be used to translate the section.',
+      },
+      forms: {
+        _type: Type.Array,
+        _description: 'Configuration of the forms.',
+        _elements: {
+          name: {
+            _type: Type.String,
+            _description: 'Name of form.',
+          },
+          uuid: {
+            _type: Type.String,
+            _description: 'Uuid of the form.',
+          },
+        },
+      },
+    },
+    _default: undefined,
+  },
+  showHtmlFormEntryForms: {
+    _type: Type.Boolean,
+    _default: true,
+    _description: 'Whether HTML Form Entry forms should be included in lists of forms.',
+  },
+  showRecommendedFormsTab: {
+    _type: Type.Boolean,
+    _description: 'Whether to display recommended forms tab. Requires AMPATH ETL.',
+    _default: false,
+  },
+  showConfigurableForms: {
+    _type: Type.Boolean,
+    _description: 'Whether to use customURL to fetch forms, works with `customFormsUrl`',
+    _default: false,
+  },
+  customFormsUrl: {
+    _type: Type.String,
+    _description: 'Custom forms endpoint to fetch forms using a custom url need `showConfigurableForms`',
+    _default: '',
+  },
+  useCurrentVisitDates: {
+    _type: Type.Boolean,
+    _description: 'Only load encounters inside of the current visits dated.',
+    _default: false,
+  },
+  orderFormsByName: {
```

Comment:
Rather than having endless numbers of boolean properties, we should try to have as few properties as possible. I'd suggest we just have an "orderBy" setting that takes a string value that can be, e.g., "name" or "most recent". This allows us to expand this option down the road...

Reply:
done